### ISSUE TITLE
Pretrained Cifar10 Weights are giving ~10% test accuracy

### ISSUE THREAD

**Issue by aravindsv** [titu1994/DenseNet#42]
I downloaded the DenseNet-40-12-Tensorflow-Backend-TF-dim-ordering.h5 file from the Releases page, compiled the model, loaded the weights, and tested it, but only got a 10.06% accuracy. Am I using the pretrained models wrong?

My code: 
```
model = densenet.DenseNet(img_dim, classes=nb_classes, depth=depth, nb_dense_block=nb_dense_block,
                          growth_rate=growth_rate, nb_filter=nb_filter, dropout_rate=dropout_rate, weights=None)
optimizer = Adam(lr=1e-3) # Using Adam instead of SGD to speed up training
model.compile(loss='categorical_crossentropy', optimizer=optimizer, metrics=["accuracy"])
print("Finished compiling")
print("Building model...")

(trainX, trainY), (testX, testY) = cifar10.load_data()

testX = testX.astype('float32')

testX = densenet.preprocess_input(testX)

Y_test = np_utils.to_categorical(testY, nb_classes)

# Load model
weights_file="weights/DenseNet-40-12-Tensorflow-Backend-TF-dim-ordering.h5"
if os.path.exists(weights_file):
    model.load_weights(weights_file, by_name=True)
    print("Model loaded.")

yPreds = model.predict(testX)
yPred = np.argmax(yPreds, axis=1)
yTrue = testY

accuracy = metrics.accuracy_score(yTrue, yPred) * 100
error = 100 - accuracy
print("Accuracy : ", accuracy)
print("Error : ", error)
```

**Comment by titu1994** [comment]
Those pretrained weights are based on a previous version of the model which was incorrect. While they still do load on this version, the output is wrong. 

I won't have the time to retrain on CIFAR

**Comment by aravindsv** [comment]
Alright, thank you. Just to confirm, is the current version of the model correct then?

**Comment by titu1994** [comment]
Yes, since it can load and evaluate the official weights properly.
